packages/axe-core-api/e2e/selenium/spec/api_spec.rb

```diff
@@ -320,4 +320,42 @@ def recursive_compact(thing)
     ft_inc = res.results.incomplete.find { |inc| inc.id == :'frame-tested' }
     expect(ft_inc).to be_nil
   end
+
+  describe "allowedOrigin" do
+    def get_allowed_origin
+      return $driver.execute_script("return axe._audit.allowedOrigins")
+    end
+
+    it "should not set when running runPartial and not legacy mode" do
+      $driver.get fixture "/index.html"
+      run_axe
+      allowed_origin = get_allowed_origin.first()
+      expect(allowed_origin).to eq "http://localhost:8000"
```

Comment:
Do you think we should be checking the `get_allowed_origin.length` here as well? Axe-core does `.push` and it might be forward-thinking to ensure that there is only one for this series of checks

Reply:
Based on the examples we should just check the 0th array https://github.com/dequelabs/axe-test-fixtures/blob/v1/README.md?plain=1#L440

Reply:
Feel free to dismiss. I see why that's the case.